### PR TITLE
Add to Home Screen button improvements

### PR DESCRIPTION
**Pull request by zans-laksa**
* Added check if browser supports app install feature
* Enabled Add to Home Screen button for tablets

### CODE REVIEW

File: src/app/component/InstallPrompt/InstallPrompt.component.js [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+        if (displayComponent) {
+            return (
+                <div block="InstallPrompt">
+                    { this.renderPrompt() }
+                </div>
+            );
+        }
+
+        return null;
```
Comment (yeegor) [inline src/app/component/InstallPrompt/InstallPrompt.component.js:69]: Inverse conditional statement logic here to reduce nesting

File: src/app/component/InstallPrompt/InstallPrompt.component.js [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+    }
+
+    render() {
+        const { device, isBannerClosed } = this.props;
+        const displayComponent = (device.ios || device.android)
+            && !device.standaloneMode
+            && !isBannerClosed
+            && this.hasSupport();
```
Comment (yeegor) [inline src/app/component/InstallPrompt/InstallPrompt.component.js:59]: Isn't this a part of the `hasSupport` method?

**Review by yeegor — CHANGES_REQUESTED** [review]
Minor coding style improvement requests

File: src/app/component/InstallPrompt/InstallPrompt.component.js (hunk 1 above)
Comment (zans-laksa) [reply src/app/component/InstallPrompt/InstallPrompt.component.js]: Done

File: src/app/component/InstallPrompt/InstallPrompt.component.js (hunk 2 above)
Comment (zans-laksa) [reply src/app/component/InstallPrompt/InstallPrompt.component.js]: Moved to `hasSupport` method.

**Review by yeegor — APPROVED** [review]
Good!

**Pull request merged**